assembly/integer/i128.ts

```diff
@@ -379,6 +447,16 @@ export class i128 {
      return result;
    }
 
+   @inline
+   i128ToU64(): u64 {
+    return u128.fromI128(this).toU64();
+  }
+
+  @inline
+  i128toi64(): i64 {
+    return this.lo; // return the lower 64 bits of x
+  }
+
```

Comment:
For such operations, we have generic `as` method. See it on `u128`:
https://github.com/MaxGraey/as-bignum/blob/master/assembly/integer/u128.ts#L840